daml-assistant/get-daml.sh

```diff
@@ -17,14 +17,17 @@
 #
 
 set -eu
+readonly SWD="$PWD"
+readonly TMPDIR="$(mktemp -d)"
+cd $TMPDIR
+
 cleanup() {
```

Comment:
Shouldn't we be using an `exit` trap for removing the temp dir? With this change, we can also remove the `rm -rf /tmp/*` from the Dockerfile again.

Reply:
We're already using an exit trap for errors ... I suppose we could have two exit traps.

Reply:
Removed `rm -rf /tmp/*` from the Dockerfile.

Reply:
Can we just put this into the already existing exit trap then?